test-it/testList.json

```diff
@@ -361,7 +361,13 @@
                 "TestVI_SimpleArrayOfRefs.via",
                 "TestVI_SpreadSheetStringToArray.via",
                 "TestVI_UnflattenFromJSONInvalidCluster.via",
```

Comment:
We should make a disabled section and place _TestVI_VariantEquality_VariantsWithVariantsNestedInsideThem.via in there if it is currently disabled